Tasks/FtpUpload/task.loc.json

```diff
@@ -31,6 +31,9 @@
   "inputs": [
     {
       "name": "credsType",
+      "aliases": [
```

Comment:
you switched to multiline here. not sure if that was desired or not

Reply:
It's the tool that generates this file.  Not me.  Do we care?